src/app/features/tech-record/components/tech-record-summary/tech-record-summary.component.ts

```diff
@@ -73,33 +75,37 @@ export class TechRecordSummaryComponent implements OnInit, OnDestroy {
     this.store
       .pipe(
         select(editableTechRecord),
-        //Need to check that the editing tech record has more than just reason for creation on and is the full object.
-        map(techRecord => {
-          if (techRecord && Object.keys(techRecord).length > 1) {
-            return cloneDeep(techRecord);
-          } else {
-            if (this.techRecord.vehicleType === VehicleTypes.HGV || this.techRecord.vehicleType === VehicleTypes.TRL) {
-              const [axles, axleSpacing] = this.axlesService.normaliseAxles(this.techRecord.axles, this.techRecord.dimensions?.axleSpacing);
-              this.techRecord = cloneDeep(this.techRecord);
-              this.techRecord.dimensions = { ...this.techRecord.dimensions, axleSpacing };
-              this.techRecord.axles = axles;
-            }
-
-            this.technicalRecordService.updateEditingTechRecord(this.techRecord, true);
-
-            return { ...cloneDeep(this.techRecord), reasonForCreation: '' };
-          }
+        skipWhile(editableTechRecord => !editableTechRecord),
+        map(editingTechRecord => {
+          return cloneDeep(editingTechRecord);
         }),
         takeUntil(this.destroy$)
       )
       .subscribe(techRecord => {
-        this.techRecordCalculated = techRecord;
-        this.referenceDataService.removeTyreSearch();
-        this.sectionTemplates = this.vehicleTemplates;
-        this.middleIndex = Math.floor(this.sectionTemplates.length / 2);
+        if (techRecord) {
+          this.techRecordCalculated = techRecord;
+          this.referenceDataService.removeTyreSearch();
+          this.sectionTemplates = this.vehicleTemplates;
+          this.middleIndex = Math.floor(this.sectionTemplates.length / 2);
+        }
       });
   }
 
+  ngOnChanges(changes: SimpleChanges): void {
+    const { techRecord } = changes;
+
+    if (techRecord.previousValue !== techRecord.currentValue && techRecord.isFirstChange()) {
```

Comment:
Very clever way of using .isFirstChange(), I applaud you sir 👏 I presume this is tested and it works.